docs/developer-guide/ksqldb-reference/create-connector.md

```diff
@@ -13,7 +13,7 @@ Synopsis
 --------
 
 ```sql
-CREATE SOURCE | SINK CONNECTOR connector_name WITH( property_name = expression [, ...]);
+CREATE SOURCE | SINK CONNECTOR connector_name [IF EXISTS] WITH( property_name = expression [, ...]);

```

Comment:
```suggestion
CREATE SOURCE | SINK CONNECTOR connector_name [IF NOT EXISTS] WITH( property_name = expression [, ...]);

```

Reply:
Ahh ! This slipped right through , fixed this